src/combat.ts

```diff
@@ -286,6 +286,10 @@ export class Macro extends StrictMacro {
           Macro.trySkill($skill`Feel Nostalgic`)
         )
       )
+      .externalIf(
+        myFamiliar() === $familiar`Space Jellyfish`,
+        Macro.if_($monster`garbage tourist`, Macro.trySkill($skill`Extract Jelly`))
```

Comment:
All tourists in the zone are stench-aligned; I think libram allows monster-arrays now, so you can use $monsters or you can use `monstername * tourist` or something like that